cxx-sensors/src/main/java/org/sonar/cxx/sensors/clangsa/CxxClangSASensor.java

```diff
@@ -88,7 +90,12 @@ protected void processReport(final SensorContext context, File report)
         NSString desc = (NSString) diag.get("description");
         String description = desc.getContent();
 
-        String checkerName = ((NSString) diag.get("check_name")).getContent();

```

Comment:
Would add for all `.get` a `Objects.requireNonNull` check (see https://github.com/SonarOpenCommunity/sonar-cxx/pull/1447/files#diff-e2fa7d302186280fa2ad45873a2294baR79)